src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/OxoGReadCounts.java

```diff
@@ -65,55 +59,37 @@ public void annotate(final ReferenceContext refContext,
         Utils.nonNull(gb, "gb is null");
         Utils.nonNull(vc, "vc is null");
 
-        if (g == null || !g.isCalled() || likelihoods == null || !vc.isSNP()) {
+        if (g == null || likelihoods == null) {
             return;
         }
 
-        final Allele ref = vc.getReference();
-        final Allele alt = vc.getAlternateAllele(0);
-
-        int alt_F1R2 = 0;
-        int alt_F2R1 = 0;
-        int ref_F1R2 = 0;
-        int ref_F2R1 = 0;
-
-        for (final ReadLikelihoods<Allele>.BestAllele bestAllele : likelihoods.bestAlleles(g.getSampleName())) {
-            final GATKRead read = bestAllele.read;
-            if (bestAllele.isInformative() && isUsableRead(read) && read.isPaired()) {
-                final Allele allele = bestAllele.allele;
-                if (allele.equals(ref, true)) {
-                    if (read.isReverseStrand() == read.isFirstOfPair()) {
-                        ref_F2R1++;
-                    } else {
-                        ref_F1R2++;
-                    }
-                } else if (allele.equals(alt, true)) {
-                    if (read.isReverseStrand() == read.isFirstOfPair()) {
-                        alt_F2R1++;
-                    } else {
-                        alt_F1R2++;
-                    }
-                }
-            }
-        }
+        final Map<Allele, MutableInt> f1r2Counts = likelihoods.alleles().stream()
```

Comment:
If you do ALT_F1R2 and ALTF2R1 counts per allele (I'm assuming it will be on a genotype), we need a test that this works for FilterByOrientationBias.  The current behavior is to ditch all but the first allele and that was done because the counts were per genotype.  I don't want to leave that hack in place if we can do the right thing.